pc2cps

```diff
@@ -113,11 +182,11 @@ def parse(path_or_name):
 
         path_or_name = out.strip()
 
-    return pkgconfig(path_or_name)
+    return pkgconfig(path_or_name, library_name, type_order)
 
 #------------------------------------------------------------------------------
 def extend(data, key, value):
-    if value is not None:
```

Comment:
What is the purpose of this change?

Reply:
Just making the code simpler. I can revert if you really dislike it. I just don't see the point of adding 3 words here.

Reply:
Upon further poking, I see it is also useful for suppressing empty lists or maps. However, as-is, this change is potentially broken, since values like `False`, `0`, that *should* be set, aren't. (Maybe that doesn't matter now, but for a generic function, I'd prefer to not have it be broken for certain inputs, even if we never give it those inputs at present.)

I have a better version locally. Please hold off on edits pending resolution of the latest general discussion.

Reply:
Ok, just let me know when you have merged your local version. I will rebased on the latest master then.